pkg/kubelet/apis/config/types.go

```diff
@@ -221,6 +227,12 @@ type KubeletConfiguration struct {
 	// TopologyManagerPolicy is the name of the policy to use.
 	// Policies other than "none" require the TopologyManager feature gate to be enabled.
 	TopologyManagerPolicy string
+	// Topology Manager Scope represents the scope of topology hint generation
```

Comment:
golint requires the first word to exactly match the name of the field

```suggestion
	// TopologyManagerScope represents the scope of topology hint generation
```

Reply:
Fixed